xla/stream_executor/rocm/rocm_blas.cc

```diff
@@ -1025,21 +962,120 @@ absl::Status ROCMBlas::DoBlasGemmBatchedInternal(
   return absl::OkStatus();
 }
 
+class rocblas_hgemm_strided_batched_mfma {
+  int ALT_;
+public:
+  rocblas_hgemm_strided_batched_mfma(int ALT) : ALT_(ALT) {}
+  static std::string kName;
+  rocblas_status operator()(rocblas_handle      handle,
+                            rocblas_operation   transA,
+                            rocblas_operation   transB,
+                            rocblas_int         m,
+                            rocblas_int         n,
+                            rocblas_int         k,
+                            const rocblas_half* alpha,
+                            const rocblas_half* A,
+                            rocblas_int         lda,
+                            rocblas_stride      stride_a,
+                            const rocblas_half* B,
+                            rocblas_int         ldb,
+                            rocblas_stride      stride_b,
+                            const rocblas_half* beta,
+                            rocblas_half*       C,
+                            rocblas_int         ldc,
+                            rocblas_stride      stride_c,
+                            rocblas_int         batch_count) {
+  float alpha32 = float(*(const __half*)alpha);
```

Comment:
possible to `static_cast` here instead? same for below lines with this casting style